CMakeLists.txt

```diff
@@ -321,31 +321,15 @@ if(CELERITAS_USE_OpenMP)
 endif()
 
 if(CELERITAS_USE_Perfetto)
-  include(FetchContent)
   if(CELERITAS_USE_CUDA OR CELERITAS_USE_HIP)
     celeritas_error_incompatible_option(
       "Perfetto is not supported with CUDA/HIP build"
       CELERITAS_USE_Perfetto
       OFF
     )
+  else()
+    celeritas_find_or_external_package(Perfetto)
```

Comment:
Should we provide a FindPerfetto module here? Otherwise, find_package will never succeed since Perfetto doesn't support CMake.

Reply:
I guess we could, is it something users even have installed externally? I'm fine with `find_package` never succeeding....

Reply:
No, it shouldn't be externally installed.  Having ```FETCHCONTENT_SOURCE_DIR_PERFETTO``` use a local checkout instead of fetching it is fine. I should have added QUIET to the find_or_external call as we get a warning for not having a module and config file.